doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+
+Install build tools and dependencies
+====================================
+On the Clear Linux build machine, install the necessary tools in order to build ACRN
+hypervisor, Service VM and Preempt-RT kernels::
+
+  $ sudo swupd bundle-add os-clr-on-clr os-core-dev python3-basic
+  $ pip3 install --user kconfiglib
+
+How to rebuild ACRN Hypervisor for industry scenario
+****************************************************
+In this section, we firstly follow :ref:`quick-setup-guide` to set up the Service VM.
+After setup, we need to rebuild acrn-hypervisor by enabling industry scenario and
+replace ``acrn.efi``.
+
+Set up the Service VM on KBL SATA Disk
+======================================
+#. Refer to the :ref:`quick-setup-guide` to boot Service VM:
+
+   .. code-block:: console
+      
+      $ export https_proxy=https://myproxy.mycompany.com:port
+      $ cd ~
+      $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/acrn_quick_setup.sh
+      $ sudo sh acrn_quick_setup.sh -s 30690
+      Password:
+      Upgrading SOS...
+      Disable auto update...
+      Clear Linux version 30690 is already installed. Continuing to setup SOS...
+      Adding the service-os, kernel-iot-lts2018 and systemd-networkd-autostart bundles...
+        ...100%
+        ...100%
+        ...100%
+      none
+      Add /mnt/EFI/acrn folder
+      Copy /usr/share/acrn/samples/nuc/acrn.conf /mnt/loader/entries/
+      Copy /usr/lib/acrn/acrn.efi to /mnt/EFI/acrn
+      Check ACRN efi boot event
+      Clean all ACRN efi boot event
+      Check linux bootloader event
+      Clean all Linux bootloader event
+      Add new ACRN efi boot event
+      Create loader.conf
+      Add default (5 seconds) boot wait time
+      Add default boot to ACRN
+      Getting latest Service OS kernel version: kernel-org.clearlinux.iot-lts2018-sos.4.19.62-73
+      Getting current Service OS kernel version: kernel-org.clearlinux.iot-lts2018-sos.4.19.13-1901141830
+      Replacing root partition uuid in acrn.conf
+      Replace with new SOS kernel in acrn.conf
+      Service OS setup done!
+      Rebooting Service OS to take effects.
+      Rebooting.
+
+#. After Service VM boots, check the ACRN status:
+
+   .. code-block:: console
+
+      $ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    1.220887] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    1.224401] ACRN HVLog: Initialized hvlog module with 4 cpu
+
+
+Build Industry scenario Hypervisor on build machine (non-root privilege)
+========================================================================
+Follow these commands to generate the ``acrn.efi`` with industry scenario::
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-hypervisor.git
+  $ cd acrn-hypervisor
+  $ git checkout acrn-2019w31.5-140000p
+  $ sudo vim hypervisor/arch/x86/Kconfig
```

Comment:
This should be done through `make menuconfig` instead.

Reply:
It's under discussing since using the ``make menuconfig`` way is not able to generate the acrn.efi for kbl board.

Reply:
OK, can we add a little note in here with a link to the issue used to track this problem?

Reply:
Now the ``make menuconfig`` is functional with the following procedure:
```
$ cd ~/acrn-hypervisor
$ make menuconfig -C hypervisor/ BOARD=kbl-nuc-i7  # select industry scenario in the first line.
$ make -C hypervisor
$ make -C misc/efi-stub HV_OBJDIR=$PWD/hypervisor/build EFI_OBJDIR=$PWD/hypervisor/build
```